src/stylesheets/elements/_tags.scss

```diff
@@ -191,6 +193,31 @@
   }
 }
 
+.sds-tag-container {
+  display: flex;
+  flex-direction: column;
+}
+
+.sds-tag--borderless {
```

Comment:
Do we need this? The tags already are styled and have borders, can we just use the nth class to remove the top border?

Reply:
> Do we need this? The tags already are styled and have borders, can we just use the nth class to remove the top border?

I think it may affect the borders of the non-stacked ones.

Reply:
You added the class, so they should be fine without it. Can you try it, it would be better not to make a class for this.

I think the issue is you aren't using sds-tag in the class list, just use that instead of the borderless class:
class="sds-tag--borderless sds-tag--outline--info-light margin-left-1